### ISSUE TITLE
Increase detection accuracy

### ISSUE THREAD

**Issue by ThomHordijk** [matterport/Mask_RCNN#1502]
Hello we are trying to use the mask rcnn model to detect and move objects. The camera we utiliz provides a grayscale imagea at 1.3MP. The objects we are trying to detect are positioned about 1 - 1.5 m's away from the camera.. We see the objects being detected reasonably however, once in every so many images the bounding box and mask are way off. We are using a dataset consiting of over 700 images. The detection confidence is about 0.999 all the time however the boxes and masks don' always cover the object properly.

We tried training with coco and imagenet. With different learning rates and weight decay values. What would be a concrete method the to increase the b-box determination.

kind regards
